scapy/contrib/automotive/bmw/hsfz.py

```diff
@@ -81,13 +89,15 @@ def __init__(self, ip='127.0.0.1', port=6801):
 
 class ISOTP_HSFZSocket(HSFZSocket):
     def __init__(self, src, dst, ip='127.0.0.1', port=6801, basecls=ISOTP):
+        # type: (int, int, str, int, Packet_metaclass) -> None
```

Comment:
Could you use `Type[Packet]` instead of `Packet_metaclass` ? You'll have to import `Type`